AWS/legos/aws_delete_ebs_snapshot/aws_delete_ebs_snapshot.json

```diff
@@ -0,0 +1,11 @@
+{
+    "action_title": "AWS Delete EBS Snapshot",
+    "action_description": "Delete EBS Snapshot for an EC2 instance",
+    "action_type": "LEGO_TYPE_AWS",
+    "action_entry_function": "aws_delete_ebs_snapshot",
+    "action_needs_credential": true,
+    "action_output_type": "ACTION_OUTPUT_TYPE_LIST",
+    "action_supports_iteration": true,
+    "action_supports_poll": true,
```

Comment:
should this have the "is remediation" line?

Reply:
@amit-chandak-unskript  said we don't need it... I've updated the PR.

Reply:
:)